src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,14 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+  let numbertoArray = [];
+  numbertoArray = number.toString().split("").reverse();
+  const resultArray = [];
+
+  for (let i = 0; i < numbertoArray.length; i++) {
+    const newNumberString = numbertoArray[i];
+    const newNumber = parseInt(newNumberString, 10)
+    resultArray.push(newNumber);
```

Comment:
As above, this does work but it can be condensed into a readable one liner without the need to create multiple variables:
```
  resultArray.push(parseInt(numbertoArray[i], 10));
```